projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-utils/case-edit.utils.ts

```diff
@@ -0,0 +1,39 @@
+
+export function convertNonASCIICharacter(character: string): string {
+  if (character === '£') {
+    // pound sign will be frequently used and works for btoa despite being non-ASCII
+    // note: this could be done for other characters provided they work for btoa()
+    return character;
+  }
+  // Note: Will convert to HTML entity
+  return CaseEditUtils.PREFIX + character.charCodeAt(0) + CaseEditUtils.SUFFIX;
+}
+
+export class CaseEditUtils {
+
+  // Note: Made these purposely odd to avoid possible data conflicts
+  public static readonly PREFIX = '&#';
+  public static readonly SUFFIX = ';';
+
+  public convertNonASCIICharacters(rawString: string): string {
+    return rawString ? rawString.replace(/[^\x20-\x7F]/g, function (c) {
```

Comment:
Sorry to be annoying, but I think that 0x7F is a control character (Delete) so it should be replaced - this should be /[^\x20-\x7E]/